turingschool/savile
# WIP: Primary Btn element - draft

#### What does this PR do?

- Updates primary button element to match buttons on Turing.io, thanks Tanner, used some code from Terminal

#### Where should the reviewer start?

- Files changed. 

#### How should this be manually tested?

- If you'd like to run locally you can see it in doc site (disclaimer I have NOT addressed the documentation yet; in another task!)
- You can also run the branch from the recent PR in Turing_io with it to see it in action

#### What are the relevant tickets?

- https://3.basecamp.com/3494409/buckets/19192671/todos/3501124293
 
#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/25447342/108867423-6986c300-75b2-11eb-976d-2ebd6de9b66a.png)

![image](https://user-images.githubusercontent.com/25447342/108867453-71466780-75b2-11eb-8874-ab8b6fcec68f.png)
